bindata/v3.11.0/openshift-apiserver/deploy.yaml

```diff
@@ -115,17 +115,23 @@ spec:
         - mountPath: /var/log/openshift-apiserver
           name: audit-dir
         livenessProbe:
-          initialDelaySeconds: 30
           httpGet:
             scheme: HTTPS
             port: 8443
             path: healthz
         readinessProbe:
-          failureThreshold: 10
```

Comment:
the default is `3`, so it will take `15s` to deem it unready. 

we also have these related settings:
```
  shutdown-delay-duration:
  - 10s # give SDN some time to converge
  shutdown-send-retry-after:
  - "true"
```
https://github.com/openshift/cluster-openshift-apiserver-operator/blob/master/bindata/v3.11.0/config/defaultconfig.yaml#L17-L20

then we should change `shutdown-delay-duration` to `20s` maybe? 15s for pod ready to be fale and then endpoint controller to remove the pod IP